### ISSUE TITLE
Incorrect capitalisation conversion in some resources in Python

### ISSUE THREAD

**Issue by jen20** [pulumi/pulumi-terraform#353]
In the GCP provider, some resources have been mapped using the Go style of all-caps for acronyms (e.g. `SSL` vs `Ssl`, `IAM` vs `Iam` and so forth). Some of the generated files in the GCP repo have incorrect naming as a result (here from `git status`)

```
?? sdk/python/bin/pulumi_gcp/compute/get_l_b_i_p_ranges.py
?? sdk/python/bin/pulumi_gcp/compute/get_v_p_n_gateway.py
?? sdk/python/bin/pulumi_gcp/compute/s_s_l_certificate.py
?? sdk/python/bin/pulumi_gcp/compute/shared_v_p_c_host_project.py
?? sdk/python/bin/pulumi_gcp/compute/shared_v_p_c_service_project.py
?? sdk/python/bin/pulumi_gcp/compute/target_s_s_l_proxy.py
?? sdk/python/bin/pulumi_gcp/compute/target_t_c_p_proxy.py
?? sdk/python/bin/pulumi_gcp/compute/u_r_l_map.py
?? sdk/python/bin/pulumi_gcp/compute/v_p_n_gateway.py
?? sdk/python/bin/pulumi_gcp/compute/v_p_n_tunnel.py
?? sdk/python/bin/pulumi_gcp/folder/i_a_m_binding.py
?? sdk/python/bin/pulumi_gcp/folder/i_a_m_member.py
?? sdk/python/bin/pulumi_gcp/folder/i_a_m_policy.py
?? sdk/python/bin/pulumi_gcp/kms/crypto_key_i_a_m_binding.py
```

**Comment by swgillespie** [comment]
Actually, these are old files from before https://github.com/pulumi/pulumi-terraform/pull/179 was fixed. Each one of these that I looked at has a properly-named file next to it. I think we just need to delete these files and everything will be fine.

TLDR: we fixed the tfgen bug a long time ago but forgot to delete the old files, I think.
